src/infrastructure/acknowledgment.rs

```diff
@@ -45,14 +45,14 @@ impl AcknowledgmentHandler {
         self.received_packets.sequence_num().wrapping_sub(1)
     }
 
-    /// Returns the ack_bitfield corresponding to which of the past 32 packets we've
+    /// Returns the `ack_bitfield` corresponding to which of the past 32 packets we've
     /// successfully received.
     pub fn ack_bitfield(&self) -> u32 {
         let most_recent_remote_seq_num: u16 = self.remote_sequence_num();
         let mut ack_bitfield: u32 = 0;
         let mut mask: u32 = 1;
 
-        // Iterate the past REDUNDANT_PACKET_ACKS_SIZE received packets and set the corresponding
+        // iterate the past `REDUNDANT_PACKET_ACKS_SIZE` received packets and set the corresponding
```

Comment:
Curious why would wouldn't capitalize these as well?

Reply:
Because there are also very small '//' comments, and those are often not a complete sentence. I do think that either way we should do everything capitalized or with a lowercase. This PR has everything lower case. It is a matter of preference.